packages/workbox-build/src/entry-points/options/defaults.js

```diff
@@ -7,18 +7,22 @@
 */
 
 module.exports = {
+  babelPresetEnvTargets: ['chrome >= 56'],
   cleanupOutdatedCaches: false,
   clientsClaim: false,
   globFollow: true,
   globIgnores: ['**/node_modules/**/*'],
   globPatterns: ['**/*.{js,css,html}'],
   globStrict: true,
   importWorkboxFrom: 'cdn',
-  injectionPointRegexp: /(precacheAndRoute\()\s*\[\s*\]\s*(\)|,)/,
+  injectionPoint: 'self.__WB_MANIFEST',
```

Comment:
Maybe `__WB_PRECACHE_MANIFEST`? So it's a little more descriptive. WDYT?

Reply:
How about `self.__WB_INJECTED_MANIFEST` to match the `injectManifest` naming?

Reply:
Hmmm, the past tense seems a bit weird to me since whenever you're reading it, it hasn't been injected yet.

Reply:
Okay, I'll just switch to` __WB_PRECACHE_MANIFEST` for now, and we can revisit if needed.